shell/client/shell.html

```diff
@@ -899,6 +902,41 @@
   </div>
 </template>
 
+<template name="adminCaps">
+  {{#if powerboxOfferUrl}}
+  <div id="powerbox-offer-popup" class="popup">
+    <h3>Powerbox Offer</h3>
+    <div>
+      <input type="text" class="autoSelect" value="{{powerboxOfferUrl}}" readonly>
+    </div>
+  </div>
+  <div id="powerbox-offer-popup-closer" class="popup-closer"></div>
+  {{/if}}
+  <p><button id="offer-ipnetwork">Offer IpNetwork Capability</button>
+     <button id="offer-ipinterface">Offer IpInterface Capability</button><br>
+    <span class="details">These buttons create IpNetwork and IpInterface capabilities respectively, and will return webkeys that you can restore in other apps. Typically these apps will be "drivers" that offer more granular protocols such as IRC or SMTP.</span></p>
```

Comment:
Maybe also: "IpNetwork represents the capability to make outbound requests to the network. IpInterface represents the capability to receive inbound connections on the machine's main network interface."